### DISCUSSION TITLE
Docker script issues

### DISCUSSION THREAD

**Discussion by Peffy95** [tteck/Proxmox#2618] [Help Needed]
Hi everyone, quite new to the scene,

I'm trying to use the docker script
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/docker.sh)"

I get no issues during the install.
When i boot up the lxc and check status of portainer and docker-compose, i think it suggests that they're not installed.
Please see below.

I'm sure it's probably an issue on my end, i've tried the script like 5 times. Also can i confirm the ports for portainer? 

Thanks

![issue](https://github.com/tteck/Proxmox/assets/152871425/1e47d48c-1de2-4d61-9e91-9ae5c3253cbb)

**Comment by aytec** [comment]
Did you try "docker compose" instead "docker-compose"?

**Reply by Peffy95** [reply]
that worked, thanks for your help and quick reply!

**Comment by tteck** [comment]
Portainer does not rely on systemd.
To verify the status of Portainer, you can use the command `docker ps --filter "name=portainer"`.

**Reply by Peffy95** [reply]
that worked too! sorry for the stupid question, really appreciate your quick response, love your work!
